common/perf-test-core/src/main/java/com/azure/perf/test/core/SleepTest.java

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.perf.test.core;
+
+import com.azure.core.util.logging.ClientLogger;
+import reactor.core.publisher.Mono;
+
+import java.time.Duration;
+import java.util.concurrent.atomic.AtomicInteger;
+
+
+class SleepTest extends PerfStressTest<PerfStressOptions> {
+    private static final ClientLogger LOGGER = new ClientLogger(SleepTest.class);
+    private static final AtomicInteger INSTANCE_COUNT = new AtomicInteger();
+    private final int secondsPerOperation;
+
+
+    public SleepTest(PerfStressOptions options) {
+        super(options);
+
+        int instanceCount = SleepTest.INSTANCE_COUNT.incrementAndGet();
+        secondsPerOperation = pow(2, instanceCount);
```

Comment:
This can be replaced by `(int) Math.pow(2, instanceCount)` and the method below can be deleted.